### ISSUE TITLE
Few warnings while building the package. 

### ISSUE THREAD

**Issue by shirishag75** [davidgfnet/whatsapp-purple#238]
Hi all, 
Dunno if the  following warnings are harmless or something needs to be fixed :-

    imgutil.c: In function ‘imgProfile’:
    imgutil.c:10:45: warning: passing argument 1 of ‘FreeImage_OpenMemory’ discards ‘const’ qualifier from pointer target type
      FIMEMORY * inimgmem = FreeImage_OpenMemory(data, size);
                                                 ^
    In file included from imgutil.c:2:0:
    /usr/include/FreeImage.h:798:52: note: expected ‘BYTE *’ but argument is of type ‘const unsigned char *’
     DLL_API FIMEMORY *DLL_CALLCONV FreeImage_OpenMemory(BYTE *data FI_DEFAULT(0), DWORD size_in_bytes FI_DEFAULT(0));
                                                        ^
    imgutil.c: In function ‘imgThumbnail’:
    imgutil.c:53:45: warning: passing argument 1 of ‘FreeImage_OpenMemory’ discards ‘const’ qualifier from pointer target type
      FIMEMORY * inimgmem = FreeImage_OpenMemory(data, size);
                                                 ^
    In file included from imgutil.c:2:0:
    /usr/include/FreeImage.h:798:52: note: expected ‘BYTE *’ but argument is of type ‘const unsigned char *’
     DLL_API FIMEMORY *DLL_CALLCONV FreeImage_OpenMemory(BYTE *data FI_DEFAULT(0), DWORD size_in_bytes FI_DEFAULT(0));

I updated after couple of weeks so would leave it to you guys to share if it's right or wrong.

**Comment by dequis** [comment]
Yeah, harmless, but wouldn't hurt fixing either.

**Comment by davidgfnet** [comment]
Pull requests always welcome! Weeping on issues is boring :D

**Comment by angelbladex** [comment]
On ArchLinux, FreeImage.h is missing, can be used installing the package freeimage. I got errors of this issue messages after recompile

**Comment by davidgfnet** [comment]
Yes, I added a new dependecy (see debian/ packaging files). So I don't think it's my problem, isn't it? :D

**Comment by angelbladex** [comment]
Today recompiled from zero and i got few messages

imgutil.c: En la función ‘imgProfile’:
imgutil.c:10:45: aviso: el paso del argumento 1 de ‘FreeImage_OpenMemory’ descarta el calificador ‘const’ del tipo del destino del puntero
  FIMEMORY * inimgmem = FreeImage_OpenMemory(data, size);
                                             ^
In file included from imgutil.c:2:0:
/usr/include/FreeImage.h:802:32: nota: se esperaba ‘BYTE *’ pero el argumento es de tipo ‘const unsigned char *’
 DLL_API FIMEMORY *DLL_CALLCONV FreeImage_OpenMemory(BYTE *data FI_DEFAULT(0), DWORD size_in_bytes FI_DEFAULT(0));
                                ^
imgutil.c: En la función ‘imgThumbnail’:
imgutil.c:53:45: aviso: el paso del argumento 1 de ‘FreeImage_OpenMemory’ descarta el calificador ‘const’ del tipo del destino del puntero
  FIMEMORY * inimgmem = FreeImage_OpenMemory(data, size);
                                             ^
In file included from imgutil.c:2:0:
/usr/include/FreeImage.h:802:32: nota: se esperaba ‘BYTE *’ pero el argumento es de tipo ‘const unsigned char *’
 DLL_API FIMEMORY *DLL_CALLCONV FreeImage_OpenMemory(BYTE *data FI_DEFAULT(0), DWORD size_in_bytes FI_DEFAULT(0));

**Comment by dequis** [comment]
Install FreeImage...

**Comment by davidgfnet** [comment]
And whats exactly the problem? You got a nice .so file at the end, right?
